bundles/org.openhab.core.io.javasound/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab-addons
```

Comment:
```suggestion
https://github.com/openhab/openhab-core
```

Reply:
Argh, shit, I didn't mean to push all these files at all as I have put the them directly into the `core.audio`bundle. Just removed them from the PR again